bin/hatch-previewer

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+'use strict'
+
+const electron = require('electron');
+const proc = require('child_process');
+
+
+const allArgs = process.argv;
+const passedArgs = allArgs.slice(2);
+
+// Start app
+var child = proc.spawn(electron,
+                       ['.', ...passedArgs],
```

Comment:
Have you tried testing this in all 3 envs?
- Within hatch-previewer
- Within l-i
- Within an ingester with previewer as a dependency

 I feel like this might not work in all of them.

Reply:
My opinion is that we don't want hatch-previewer within libingester anymore, since part of the reason for this split is to lose the heavy electron dependency for every ingester. So I don't think the latter two are still use cases that need to be supported.

Reply:
Even if you exclude that, let's say you have a dev dep on this lib in an ingester - can the new code work fine in that case when you invoke `npm previewer` or `node_modules/.bin/hatch-preview` with a relative path (I can't tell for sure but I'd say it's unlikely)?

The use case here is that you can both preview a hatch from this repo itself using relative/absolute paths but also preview hatches from one of the ingester directories using also relative or absolute paths invoking this script in the `node_modules/.bin/` directory.

Reply:
I don't understand the use case for having hatch-previewer as a dev dependency after this, since hatch-previewer and libingester aren't tied together. Wouldn't it be better to install it with `npm install -g` or as a flatpak?

Reply:
On re-examining the code, I still don't see why this is needed. It looks up the path of the libingester module that hatch-previewer is part of, but this module isn't part of libingester anymore. If it's not part of libingester, then we don't need to change the CWD. If we don't change the CWD, then we don't need to resolve a relative path passed on the command line.